src/NServiceBus.Gateway/NServiceBus.Gateway.csproj

```diff
@@ -11,6 +11,14 @@
     <Reference Include="System.Web" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Remove="obsoletes-v4.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="obsoletes-v4.cs" />
+  </ItemGroup>
+
```

Comment:
Did VS do something weird here? I think we can remove these lines?

```suggestion
```

Reply:
Yea, fixed